App/Style/Animations/AnimationView+Utils.swift

```diff
@@ -54,6 +54,7 @@ enum AnimationAsset: String, CaseIterable, Equatable {
   case welcome2 = "welcomeSecond"
   case welcome3 = "welcomeThird"
   case welcome4 = "welcomeFourth"
+  case cardFlagEuropa
```

Comment:
Please add a raw value here to prevent potential refactors from breaking the code